lib/sqitch-rebase.pod

```diff
@@ -63,6 +65,13 @@ L<sqitchchanges> for the various ways in which changes can be specified.
 Specify the deployment change. Defaults to the last point in the plan. See
 L<sqitchchanges> for the various ways in which changes can be specified.
 
+=item C<--revised>
+
+Finds the change to revert onto based on the output of L<sqitch-check>. The
```

Comment:
Not sure this is accurate: `based on the output of L<sqitch-check>`. Is there output from `sqitch check` when you run this? I thought it was simply that it would rebase on the last change that does not conflict between the plan and the deployed changes. I think you can omit the part about the output of sqitch check from this description.